open_lm/norms.py

```diff
@@ -55,7 +67,16 @@ def reset_parameters(self) -> None:
                 self.bias.zero_()
 
     def forward(self, input: Tensor) -> Tensor:
-        return F.layer_norm(input, self.normalized_shape, self.weight, self.bias, self.eps)
+        if using_te:
```

Comment:
@achalddave do u think we should have a seperate class for TeLayerNorm? or do u prefer combining it with existing layer norm

Reply:
We can have a separate class for TeLayerNorm.